harness/tests/src/main/kotlin/godot/tests/inheritance/AbstractClassInheritanceEmptyParent.kt

```diff
@@ -0,0 +1,6 @@
+package godot.tests.inheritance
+
+import godot.KinematicBody
+
+// issue: https://github.com/utopia-rise/godot-kotlin-jvm/issues/365
+abstract class AbstractClassInheritanceEmptyParent: KinematicBody()
```

Comment:
is the missing `@RegisterClass` intentional?

Reply:
Yes. Abstract classes don't need to be registered. See: https://godot-kotl.in/en/stable/user-guide/advanced/abstract_classes/